change/react-native-windows-extended-2020-02-13-14-37-32-vs2019.json

```diff
@@ -1,9 +0,0 @@
-{
-  "type": "prerelease",
-  "comment": "Switching to VS 2019 and v142 tools",
```

Comment:
Revert this? Guessing it hasn't been deleted yet since no publish?

Reply:
react-native-windows-extended was deleted... so this is dead.  beachball will never delete it because it will never complete the publish of react-native-window-extended.